doc/api/os.md

```diff
@@ -1160,6 +1160,38 @@ The following error codes are specific to the Windows operating system:
   </tr>
 </table>
 
+### dlopen Constants
+
+The following signal constants are exported by `os.constants.dlopen`.
+See `man 3 dlopen` for detailed informatio.
+
+<table>
+  <tr>
+    <th>Constant</th>
+  </tr>
+  <tr>
+    <td><code>RTLD_LAZY</code></td>
```

Comment:
I thought the descriptions were too detailed and complex, and was expecting the reader could refer to the manual instead? It's a 1:1 mapping of the `dlopen` flags, so there's not any secret sauce.